README.md

```diff
@@ -67,24 +83,29 @@ genotype matrix, and the phenotype vector.
   included manually, if desired.
 
 - `weights` encodes the relative importance of BMVs, DMVs, and PTVs. The
-  example weights of `c(1, 2, 3)` target a genetic architecture where effect sizes increase with increasing deleteriousness:
-  BMVs have an effect of 1, DMVs have an effect of 2, and PTVs have an effect of 3. Weights of
-  `c(1, 1, 1)` target instead a genetic architecture where all variant
-  types have equivalent expected magnitudes.
+  example weights of `c(1, 2, 3)` target a genetic architecture where
+  effect sizes increase with increasing deleteriousness: BMVs have an
+  effect of 1, DMVs have an effect of 2, and PTVs have an effect of 3.
+  Weights of `c(1, 1, 1)` target instead a genetic architecture where
+  all variant types have equivalent expected magnitudes.
 
 ``` r
 show(results)
 ```
 
-    ##        p_count          p_ind    p_max_count      p_max_ind    p_sum_count 
-    ##   7.707024e-29   6.745269e-06   4.299938e-13   6.228669e-07   6.756953e-18 
-    ##      p_sum_ind p_allelic_skat         p_omni 
-    ##   1.894500e-06   8.507977e-08   9.248429e-28
+    ##          n_bmv          n_dmv          n_ptv        p_count          p_ind 
+    ##   2.870000e+02   1.620000e+02   6.100000e+01   3.112702e-26   1.322084e-09 
+    ##    p_max_count      p_max_ind    p_sum_count      p_sum_ind p_allelic_skat 
+    ##   3.076876e-10   5.374363e-09   1.661854e-20   2.554417e-11   2.658137e-07 
+    ##         p_omni 
+    ##   3.735235e-25
```

Comment:
I looked into this: I think there is currently a mismatch between the data and the p-values in the README (likely because `cache = TRUE` was on). When I reinstalled the previous version of the package and regenerated the p-values from the README, the results are consistent with the newer version.

**Results with v0.0.4.1**
<img width="534" alt="v0 4 1" src="https://github.com/insitro/AllelicSeries/assets/91571107/0db089ff-797d-4654-85db-a68771405d17">

**Results with v0.0.5.0**
<img width="534" alt="v0 0 5 0" src="https://github.com/insitro/AllelicSeries/assets/91571107/fe08974c-b752-4ac2-965f-3eb2ef2bfab9">

Reply:
Ah ok great - thanks for digging in. LGTM in that case